rasa/cli/data.py

```diff
@@ -68,8 +70,26 @@ def split_nlu_data(args):
 
     data_path = get_validated_path(args.nlu, "nlu", DEFAULT_DATA_PATH)
     data_path = data.get_nlu_directory(data_path)
+
     nlu_data = load_data(data_path)
+    fformat = get_file_format(data_path)
+
     train, test = nlu_data.train_test_split(args.training_fraction)
 
-    train.persist(args.out, filename="training_data.json")
-    test.persist(args.out, filename="test_data.json")
+    train.persist(args.out, filename="training_data.{}".format(fformat))
+    test.persist(args.out, filename="test_data.{}".format(fformat))
+
+
+def get_file_format(resource_name: Text) -> Text:
+    files = list_files(resource_name)
```

Comment:
there is a helper in `rasa.nlu.training_data.loading` called `_guess_format`, I think we should use the same code everywhere